lib/pseudos.js

```diff
@@ -262,7 +269,24 @@ var pseudos = {
     link: function(elem, adapter) {
         return adapter.hasAttrib(elem, "href");
     },
-    visited: falseFunc, //Valid implementation
+    hover: function(elem, adapter) {
+        if (adapter.isHovered) {
```

Comment:
I am checking this here also in case `pseudo` function is being used directly instead of from `compile`.

As we are exporting all `pseudos`, third party can use this function directly also.

Reply:
My preference would be to implement these as filters, which can return `falseFunc` themselves.